packages/dx-react-scheduler/src/plugins/drag-drop-provider.tsx

```diff
@@ -312,13 +314,43 @@ class DragDropProviderBase extends React.PureComponent<
         </Template>
 
         <Template name="allDayPanel">
-          <TemplatePlaceholder />
-          {renderAppointmentItems(this.allDayDraftAppointments, Container, draftData)}
+          <TemplateConnector>
+            {({ currentView, groupOrientation }) => {
+              if (groupOrientation?.(currentView.name) === VERTICAL_GROUP_ORIENTATION) {
+                return <TemplatePlaceholder />;
+              }
+
+              return (
+                <>
+                  <TemplatePlaceholder />
+                  {renderAppointmentItems(this.allDayDraftAppointments, Container, draftData)}
+                </>
+              );
+            }}
+          </TemplateConnector>
         </Template>
 
         <Template name="timeTable">
-          <TemplatePlaceholder />
-          {renderAppointmentItems(this.timeTableDraftAppointments, Container, draftData)}
+          <TemplateConnector>
+            {({ currentView, groupOrientation }) => {
+              if (groupOrientation?.(currentView.name) === VERTICAL_GROUP_ORIENTATION) {
+                return (
+                  <>
+                    <TemplatePlaceholder />
+                    {renderAppointmentItems(this.allDayDraftAppointments, Container, draftData)}
+                    {renderAppointmentItems(this.timeTableDraftAppointments, Container, draftData)}
+                  </>
+                );
+              }
```

Comment:
Instead of `if` use ternary operator

```jsx
<>
  <TemplatePlaceholder />
  {renderAppointmentItems(this.allDayDraftAppointments, Container, draftData)}
  {groupOrientation?.(currentView.name) === VERTICAL_GROUP_ORIENTATION && renderAppointmentItems(this.timeTableDraftAppointments, Container, draftData)}
</>
```

And same places too.